docs/apache-airflow-providers-amazon/connections/aws.rst

```diff
@@ -31,10 +31,12 @@ Authentication may be performed using any of the `boto3 options <https://boto3.a
 To use IAM instance profile, create an "empty" connection (i.e. one with no Login or Password specified, or
 ``aws://``).
 
+
 Default Connection IDs
 -----------------------
 
-The default connection ID is ``aws_default``.
+The default connection ID is ``aws_default``. If the environment/machine where you are running Airflow has the
+file credentials in ``/home/.aws/``, the default connection will take automatically the credentials from there.
```

Comment:
This is only true so long as the user/pass is empty isn't it? If so we should explicitly say that.